tests/test_isrFunctions.py

```diff
@@ -51,6 +51,98 @@ def computeImageMedianAndStd(image):
     return (median, std)
 
 
+def setExposureSatCore(exposure, x, y, halfWidthX, halfWidthY,
```

Comment:
If this is an internal helper function without a docstring, put a leading underscore on the name.  (Even so it should have a docstring.  It'll help 6 months from now!)